SetupBLT.cmake

```
@@ -33,6 +33,12 @@ if (NOT BLT_LOADED)
     # Show CMake info right out of the gate
     ################################
     message(STATUS "CMake Version: ${CMAKE_VERSION}")
+    if (DEFINED CMAKE_BUILD_TYPE)
+        message(STATUS "CMake Build Type: ${CMAKE_BUILD_TYPE}")
+    endif()
+    if (DEFINED CMAKE_CONFIGURATION_TYPES)
+        message(STATUS "CMake Configuration Types: ${CMAKE_CONFIGURATION_TYPES}")
+    endif()
```

Suggestion: Would it make sense to check the `GENERATOR_IS_MULTI_CONFIG` property to see which one is expected? Looks like this has been available since version CMake@3.9.

E.g. something like:
```suggestion
    get_property(_is_multi_config GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
    if(_is_multi_config)
        message(STATUS "CMake Configuration Types: ${CMAKE_CONFIGURATION_TYPES}")
    else()
        message(STATUS "CMake Build Type: ${CMAKE_BUILD_TYPE}")
    endif()
    unset(_is_multi_config)
```

See: https://cmake.org/cmake/help/latest/prop_gbl/GENERATOR_IS_MULTI_CONFIG.html

---

good to know! just changed it and tested it (for single target generator only though). seems to work fine.

---

Thanks @chapman39 !